src/bioregistry/data/bioregistry.json

```
@@ -60095,8 +60096,9 @@
       "prefix": "lbo"
     },
     "pattern": "^\\d{7}$",
+    "rdf_uri_format": "http://purl.obolibrary.org/obo/LBO_$1",
     "repository": "https://github.com/AnimalGenome/livestock-breed-ontology",
-    "uri_format": "http://purl.obolibrary.org/obo/LBO_$1"
+    "uri_format": "https://bioportal.bioontology.org/ontologies/LBO/?p=$1"
```

This doesn't seem to actually link to the specific entry, the right URL would be this, I think:

```suggestion
    "uri_format": "https://bioportal.bioontology.org/ontologies/LBO/?p=classes&conceptid=http://purl.obolibrary.org/obo/LBO_$1"
```